docs/6/6.8-eks.md

```diff
@@ -91,12 +91,13 @@ In the previous section we launch the SockShop demo on an ECS cluster to demonst
 1. Create IAM roles to allow AWS to manage EKS services
   - You will need a role to allow EKS to manage clusters
   - You will need another role to allow EC2 instances to call EKS services(will need to refer to CloudFormation script from AWS tutorial)
-2. Install the `aws-iam-authenticator` following these [instructions](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html)
-3. Create an EKS cluster
-4. Connect your `kubectl` tool to your cluster
-5. Create a Launch Configuration and Autoscaling group to launch Kubernetes EC2 images
-6. Connect your EC2 instances to your cluster
-7. Launch SockShop application on your cluster using the `complete-demo.yaml` in the [microservice-demo](https://github.com/liatrio/microservices-demo) repo
+2. Create an EKS cluster
+3. Connect your `kubectl` tool to your cluster
+4. Create a Launch Template with a Kubernetes optimized AMI. The instance type should be T3.medium because Kubernetes needs more resources
```

Comment:
I think this could be another note at the bottom. I would also like people to have to look at how big different EC2 types are and try to decide what they need instead of automatically select t3.medium.

Maybe something like
```
?> Make sure to use an EC2 instance type with enough CPU and memory resources for the each nodes overhead and the pods workload (t2.micro will not be enough). [EC2 instance types](https://aws.amazon.com/ec2/instance-types/)
```

Reply:
Sounds good, added that